Workbooks/Virtual Machines - Network Dependencies/Connections Overview/Connections Overview.workbook

```diff
@@ -52,8 +52,8 @@
             "version": "KqlParameterItem/1.0",
             "name": "ComputerName",
             "type": 1,
-            "isRequired": true,
-            "query": "print '{Computer:label}'",
+            "isRequired": false,
+            "query": "print 'undefined'",
```

Comment:
I thought you were planning on creating a different workbook for single VM. Is that not the case?

Reply:
Yes, a separate workbook for Single VM exists, it's just that the cleanup in this current workbook hasn't been fully completed. Will do this in a future commit. Thanks.